README.md

```diff
@@ -101,6 +101,7 @@ like. Control service-specific configuration examples are
 [ir-control.yaml](config/example/ir-control.yaml) and
 [node-control.yaml](config/example/node-control.yaml) for IR and SN nodes
 respectively.
+For information about CLI exit codes and their meanings, refer to [CLI Return Codes](docs/cli-return-codes.md).
```

Comment:
I think this should be added into the "some specific concepts have additional documents describing them" list above, like ` * [Exit codes](docs/...)`